tnsnames.ora

```diff
@@ -0,0 +1,161 @@
+
+
+
+CMS_PXL =
```

Comment:
why is this file `tnsnames.ora` needed?

Reply:
we should not distribute this file. CERN IT does not like it :-(

Reply:
Thanks, I removed it